docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+
+### Into other apps
+
+Linking to other apps from your app is achieved using a URL based on the target app's URL scheme. This **URL scheme** allows you to reference resources within that native app.
+
+Your app can use a [common URL scheme](/linking/into-other-apps/#common-url-schemes) for default apps or implement any URL scheme, including `https` and `http` (commonly used by web browsers like Chrome, Safari, and so on), and use JavaScript to handle the URL that launches the corresponding native app.
+
+<BoxLink
+  title="Linking into other apps"
+  description="Learn how to handle common and custom URL schemes to link other apps from your app."
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+### Into your app
+
+Linking to your app from other apps or a web browser is achieved using a URL based on the scheme for your app. These URLs are called **deep links**.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
+### Android App Links
+
+Android App Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to Android devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website.
+
+<BoxLink
+  title="Configure Android App Links"
+  description={
+    <>
+      Learn how to configure <CODE>intentFilters</CODE> and set up two-way association from a
+      standard web URL.
+    </>
+  }
+  href="/linking/android-app-links/"
+  Icon={AndroidIcon}
+/>
+
+### iOS Universal Links
+
+iOS Universal Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to iOS devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website. You can further configure the website by displaying a banner to the user to open your app using [Apple Smart Banner](/linking/ios-universal-links/#apple-smart-banner).
+
+<BoxLink
+  title="Configure iOS Universal Links"
+  description={
+    <>
+      Learn how to configure <CODE>associatedDomains</CODE> and set up two-way association.
+    </>
+  }
+  href="/linking/ios-universal-links/"
+  Icon={AppleIcon}
+/>
+
+## Use Expo Router to handle deep linking
+
+To implement any of the above Linking strategy, **we recommend using** [Expo Router](/router/introduction/) since deep linking is automatically enabled for all of your app's screens.
```

Comment:
```suggestion
To implement any of the above Linking strategies, **we recommend using** [Expo Router](/router/introduction/) since deep linking is automatically enabled for all of your app's screens.
```